fashion-messaging-app-ui/src/App.jsx

```diff
@@ -41,7 +41,7 @@ export default function App() {
         console.error("Error fetching photos:", error);
       });
       localStorage.setItem('user', JSON.stringify(user));
-  },[user], [searchQuery]);
```

Comment:
I removed the user from the useEffect hook because it was making my search button unable to listen for clicks and I wasn't getting any error. I took it out and my search worked fine. I still don't know why but would like to go over this tomorrow